telegram/helpers.py

```diff
@@ -44,23 +44,28 @@
 def escape_markdown(text: str, version: int = 1, entity_type: str = None) -> str:
     """Helper function to escape telegram markup symbols.
 
+    .. versionchanged:: NEXT.VERSION
+        Custom emoji entity escaping is now supported.
+
     Args:
         text (:obj:`str`): The text.
         version (:obj:`int` | :obj:`str`): Use to specify the version of telegrams Markdown.
             Either ``1`` or ``2``. Defaults to ``1``.
         entity_type (:obj:`str`, optional): For the entity types
             :tg-const:`telegram.MessageEntity.PRE`, :tg-const:`telegram.MessageEntity.CODE` and
-            the link part of :tg-const:`telegram.MessageEntity.TEXT_LINK`, only certain characters
+            the link part of :tg-const:`telegram.MessageEntity.TEXT_LINK` and
+            :tg-const:`telegram.MessageEntity.CUSTOM_EMOJI`, only certain characters
             need to be escaped in :tg-const:`telegram.constants.ParseMode.MARKDOWN_V2`.
-            See the official API documentation for details. Only valid in combination with
-            ``version=2``, will be ignored else.
+            See the `official API documentation <https://core.telegram.org/bots/api
+            #formatting-options>`__ for details. Only valid in
+            combination with ``version=2``, will be ignored else.
     """
     if int(version) == 1:
         escape_chars = r"_*`["
     elif int(version) == 2:
         if entity_type in ["pre", "code"]:
             escape_chars = r"\`"
-        elif entity_type == "text_link":
+        elif entity_type in ["text_link", "custom_emoji"]:
```

Comment:
I don't see this in the documentation 🤔  Can a `)` even be part of the `tg://emoji?id=<numeric>` thingy?

Reply:
_Inside the (...) part of the inline link and custom emoji definition, all ')' and '\' must be escaped with a preceding '\' character._

:Shrug:

Reply:
Hah. I'm blind. Thanks :)